ui/page/root/wallet_list.go

```diff
@@ -358,13 +357,21 @@ func (pg *WalletSelectorPage) listenForNotifications() {
 			return
 		}
 
+		// TODO: Probably also need to register a tx and block listener to
+		// update wallet balances when the wallet receives new txs.
+
 		go func(wal sharedW.Asset) {
 			for {
 				select {
 				case n := <-syncListener.SyncStatusChan:
 					if n.Stage == wallet.SyncCompleted {
+						// TODO: Does this update the wallet balances??
+						// pg.loadWallets() is only called when the page is
+						// about to be displayed, not when it the window is
+						// reloaded.
 						pg.ParentWindow().Reload()
```

Comment:
A bigger question maybe, is, do we need to cache the wallet balances? If we read balance from each wallet directly instead of caching, just reloading here would do.